www/caddy/src/opnsense/mvc/app/controllers/OPNsense/Caddy/ReverseProxyController.php

```diff
@@ -38,11 +38,49 @@ class ReverseProxyController extends IndexController
     public function indexAction()
     {
         $this->view->pick('OPNsense/Caddy/reverse_proxy');
-        $this->view->formDialogReverseProxy = $this->getForm("dialogReverseProxy");
-        $this->view->formDialogSubdomain = $this->getForm("dialogSubdomain");
-        $this->view->formDialogHandle = $this->getForm("dialogHandle");
-        $this->view->formDialogAccessList = $this->getForm("dialogAccessList");
-        $this->view->formDialogBasicAuth = $this->getForm("dialogBasicAuth");
-        $this->view->formDialogHeader = $this->getForm("dialogHeader");
+
+        $formDialogReverseProxy = $this->getForm("dialogReverseProxy");
+        $this->view->formDialogReverseProxy = $this->preprocessFields($formDialogReverseProxy, 'reverse.');
+
+        $formDialogSubdomain = $this->getForm("dialogSubdomain");
+        $this->view->formDialogSubdomain = $this->preprocessFields($formDialogSubdomain, 'subdomain.');
+
+        $formDialogHandle = $this->getForm("dialogHandle");
+        $this->view->formDialogHandle = $this->preprocessFields($formDialogHandle, 'handle.');
+
+        $formDialogAccessList = $this->getForm("dialogAccessList");
+        $this->view->formDialogAccessList = $this->preprocessFields($formDialogAccessList, 'accesslist.');
+
+        $formDialogBasicAuth = $this->getForm("dialogBasicAuth");
+        $this->view->formDialogBasicAuth = $this->preprocessFields($formDialogBasicAuth, 'basicauth.');
+
+        $formDialogHeader = $this->getForm("dialogHeader");
+        $this->view->formDialogHeader = $this->preprocessFields($formDialogHeader, 'header.');
```

Comment:
A bit too much boilerplate for my taste, but I'm sure this can be improved. Can't the form hold the additional argument too?

Reply:
My main issue here was this:

https://github.com/opnsense/plugins/blob/dbe996b0539d9e8b4627354527f1281cc97aef5d/www/caddy/src/opnsense/mvc/app/views/OPNsense/Caddy/reverse_proxy.volt#L353

`data-column-id="enabled"`

https://github.com/opnsense/plugins/blob/dbe996b0539d9e8b4627354527f1281cc97aef5d/www/caddy/src/opnsense/mvc/app/controllers/OPNsense/Caddy/forms/dialogReverseProxy.xml#L3

`<id>reverse.enabled</id>`

I need to split the id. I could not process that in the template due to restrictions.

Maybe this boilerplate can be centralized at a different spot so I can always receive a `data-column-id` that has been generated from the `id`.